chain/bitcoind_client.go

```diff
@@ -235,7 +235,7 @@ func (c *BitcoindClient) LoadTxFilter(reset bool,
 	// If we reset, signal that.
 	if reset {
 		select {
-		case c.rescanUpdate <- reset:
+		case c.rescanUpdate <- struct{}{}:
```

Comment:
What's the diff?

Reply:
The event loop was previously not handling receiving `bool` as you can see in https://github.com/btcsuite/btcwallet/blob/master/chain/bitcoind.go#L570.

Reply:
I believe so yeah 😅